eng/packages/http-client-csharp-mgmt/generator/TestProjects/Local/Mgmt-TypeSpec/src/Generated/MgmtTypeSpecTestsModelFactory.cs

```diff
@@ -64,8 +64,8 @@ public static FooData FooData(ResourceIdentifier id = default, string name = def
                     boolValue,
                     floatValue,
                     doubleValue,
-                    prop1,
-                    prop2,
+                    prop1.ToList(),
+                    prop2.ToList(),
```

Comment:
I think we should have something like `prop1 ??= new ChangeTrackingList` stuff in this method like other normal lists?